### DISCUSSION TITLE
Pickup version from monorepo with multiple versions

### DISCUSSION THREAD

**Discussion by beatkind** [renovatebot/renovate#20764] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.97.5

### If you're self-hosting Renovate, select which platform you are using.

github.com

### If you're self-hosting Renovate, tell us what version of the platform you run.

latest

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We want to have Renovate pickup GitHub releases that are named like _modulename_-v0.8.1 and update these. The source repo is a mono terraform modules repo. The source repo contains multiple releases:

- _modulename_-v0.8.1
- _modulename2_-v1.1.0
- _modulename3_-v0.5.8

### What you tried so far.

We did not find something that would match our situation or have overseen such a config option.

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Dependency github.com/retracted has unsupported value modulname1-v0.8.0 (repository=retracted)

```

</details>

**Comment by rarkins** [comment]
Check out the docs and other discussions for `extractVersion`
